### PR TITLE
adds posts post threshold (time) to settings

### PR DESCRIPTION
**Pull request by converge**
- minor adjustment to remove the hardcoded hour threshold/limit.
- I did identify a bug in how we're collecting the post date, going to fix it on next PR.

### CODE REVIEW

File: instapy/settings.py [hunk 1]
```diff
@@ -80,6 +80,10 @@ class Settings:
     # This is where currently the pods server is hosted
     pods_server_endpoint = 'https://us-central1-instapy-pods.cloudfunctions.net'
 
+    # Hour threshold to upload Pods Post ID's
+    pods_active_post_hour_limit = 12
```
Comment (ishandutta2007) [inline instapy/settings.py:84]: Lets make it 2 or 3 hours, that's what instagram's algorithm uses to decide if a post needs to go viral or not. Initially we had set it high as we were not sure how much traffic we may get, but now we are getting hundreds of post each hour.

File: instapy/instapy.py [hunk 2]
```diff
@@ -5310,20 +5310,36 @@ def join_pods(self, topic='general', engagement_mode='normal'):
                 try:
                     web_address_navigator(self.browser, post_link)
                     sleep(2)
+                    # FIXME: datetime HTML TIME element is different
```
Comment (ishandutta2007) [inline instapy/instapy.py:5313]: Yes I noticed this long back. For 12.5 hour window it wasn't a big issue, when we reduce the window we will need to use local time.

**Review by ishandutta2007 — CHANGES_REQUESTED** [review]

**Comment by converge** [comment]
going to update this PR soon, please don't merge it just yet.

**Comment by breuerfelix** [comment]
please remove the tag when we are ready to merge it ! so its easier to know which PRs are ready to merge when having a look at the PR list :)

**Comment by ishandutta2007** [comment]
@converge where are we on this, I thought you said you will address the timezone issue and resubmit?

**Comment by converge** [comment]
hey @ishandutta2007 , sorry for the delay, I'll postpone it, I have been working to make instapy more stable for now. I will reach it soon.

**Comment by ishandutta2007** [comment]
Lets merge the rest then ? Create a separate issue to track the timezone issue ?

**Pull request closed without merging**